### PR TITLE
UI Update

### PR DESCRIPTION
**Pull request by Ameya64**

### CODE REVIEW

**Review by zcam007 — APPROVED** [review]

**Review by zcam007 — APPROVED** [review]

**Review by zcam007 — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,15 +6,15 @@
     <string name="last_name">Last Name</string>
     <string name="login_title">Login</string>
     <string name="create_act">Create Account</string>
-    <string name="login_password">Enter your password here</string>
+    <string name="login_password">********</string>
```
Comment (zcam007) [inline app/src/main/res/values/strings.xml:9]: Is this part of password hint?

**Review by zcam007 — CHANGES_REQUESTED** [review]

**Review by zcam007 — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Ameya64) [reply app/src/main/res/values/strings.xml:9]: Yeah bro, it's like a demo entry, to make it look a little modern, if you want I'll change it back

Comment (zcam007) [reply app/src/main/res/values/strings.xml:9]: No, it's fine. Just asked to know if they are part of Password hint. Seems good👍

Comment (Ameya64) [reply app/src/main/res/values/strings.xml:9]: Thanks bro

**Pull request merged**